src/Microsoft.Framework.DependencyInjection.Abstractions/FrameworkServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Framework.DependencyInjection.Abstractions;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection.FrameworkExtensions
```

Comment:
Maybe just `Extensions` for now? I don't like that either, but I dislike it less :smile: